common.go

```diff
@@ -0,0 +1,7 @@
+package scalingo
```

Comment:
nitpick picky: I think `owner.go` is a better name ;)

> Packages named util, common, or misc provide clients with no sense of what the package contains

https://go.dev/blog/package-names#bad-package-names

I think the same of file name